src/sentry/db/models/fields/array.py

```diff
@@ -41,12 +38,7 @@ def to_python(self, value):
         return map(self.of.to_python, value)
 
     def get_db_prep_value(self, value, connection, prepared=False):
-        if not prepared:
-            engine = connection.settings_dict["ENGINE"]
-            if "postgres" in engine:
-                return value
-            return json.dumps(value) if value else None
-        return value
+        return super(ArrayField, self).get_db_prep_value(value, connection, prepared=prepared)
```

Comment:
I can just remove this altogether, right?

Reply:
Looks like it, since we always just return `value` for postgres